packer/vagrant-box.json

```diff
@@ -151,7 +236,25 @@
                 "scripts/install-certstrap.sh",
                 "scripts/usr-local-bin-to-path.sh",
                 "scripts/create-kube-certs.sh",
-                "scripts/create-kubedns.sh",
+                "scripts/create-kubedns.sh"
+            ],
+            "start_retry_timeout": "7m"
+        },
+        {
+            "type": "shell",
+            "only": ["triton"],
+            "inline": [
+                "# Install helm",
+                "wget -O - https://kubernetes-helm.storage.googleapis.com/helm-v2.5.1-linux-amd64.tar.gz | tar xz -C /usr/local/bin --no-same-owner --strip-components=1 linux-amd64/helm",
```

Comment:
Since we're already breaking parity with the other images built, I wonder if it would make sense to have two triton image targets... one for CI and one for development.. the CI one could use `bin/common/install_tools.sh` to install helm, k, and cf. The dev target could additionally provision the image with `bin/dev/install_tools.sh`

Reply:
Unfortunately, I don't think that's possible: the target name is the builder type.

Reply:
My preference would be to create a `triton-dev.json` config file then, which takes the image produced from the regular CI triton build, and adds in the dev tools.

Alternatively, we could just throw all the tools into the image, or require people using the same image for development to install them themselves.

I dislike all of these options though :(

Reply:
Yeah, I don't really like any of the options either.  Hence taking the easy option for now until we get better ideas…

Reply:
We'll probably want to track the need for the triton dev image packer config, but this will work for CI